projects/aca-testing-shared/src/components/dialog/create-from-template-dialog.ts

```diff
@@ -31,9 +31,9 @@ export class CreateFromTemplateDialog extends GenericDialog {
   createButton = this.childElement(by.cssContainingText('.mat-dialog-actions button', 'Create'));
   cancelButton = this.childElement(by.cssContainingText('.mat-dialog-actions button', 'CANCEL'));
 
-  nameInput = this.childElement(by.css('input[placeholder="Name" i]'));
-  titleInput = this.childElement(by.css('input[placeholder="Title" i]'));
-  descriptionTextArea = this.childElement(by.css('textarea[placeholder="Description" i]'));
+  nameInput = this.rootElem.element(by.css('input[data-placeholder="Name" i]'));
```

Comment:
I know it's not the change by this PR, but selecting elements by placeholders is completely wrong as for me. Maybe we have separate task to use proper test ids?

Reply:
yep maybe is better do it in a separate PR